plugins/microsoft_teams/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: microsoft_teams
 title: Microsoft Teams
 description: The Microsoft Teams plugin allows you to send and trigger workflows on new messages. The plugin will also allow for teams management with the ability to add and remove teams, channels, and users
-version: 4.1.0
+version: 4.2.0
```

Comment:
With the role field now newly required for adding a member to channel, is this non backwards compatible and if so should the major version be updated? e.g. 5.0.0 ?

Reply:
It's like a new functionality, where the connection part is not changed at all. It's backward compatible with all the actions, and connection part except `Add User to Channel` action 😄 . The role field has it's own `default` value so it should not affect at all.

Reply:
ok great, thanks for the explanation, sounds like 4.2.0 should be fine then :)